docs/source/datasets/index.rst

```diff
@@ -369,8 +369,8 @@ The following columns are included in this dataset:
    * - First name
      - :code:`first_name`
      -
-   * - Middle initial
-     - :code:`middle_initial`
+   * - Middle name
```

Comment:
I think we need to add information on the noising page about how this column (which hasn't previously appeared in any dataset) is noised.

Reply:
Correct, we need to update the noise page as well. Basically we want to apply the same noise types as first name, and use the same fake names as for first name. See the concept model PR https://github.com/ihmeuw/vivarium_research/pull/1294 for details.